packages/thirdweb/src/extensions/lens/read/resolveAddress.ts

```diff
@@ -0,0 +1,80 @@
+import type { Chain } from "../../../chains/types.js";
+import type { ThirdwebClient } from "../../../client/client.js";
+import { getAddress, isAddress } from "../../../utils/address.js";
+
+/**
+ * @extension LENS
+ */
+export type ResolveLensAddressParams = {
+  handleOrLocalName: string;
+  client: ThirdwebClient;
+  overrides?: {
+    lensHandleAddress?: string;
```

Comment:
lensHandleContractAddress i assume? otherwise its confusing.

can you also add ts doc to each prop?